cpp/ql/test/query-tests/Likely Bugs/Memory Management/ReturnStackAllocatedMemory/test.cpp

```diff
@@ -248,4 +248,9 @@ char* test_strdupa(const char* s) {
 void* test_strndupa(const char* s, size_t size) {
 	char* s2 = strndupa(s, size);
 	return s2; // BAD
-}
\ No newline at end of file
+}
+
+UNKNOWN_TYPE test_error_type() {
+	UNKNOWN_TYPE x;
+	return x; // GOOD: Don't report error types
+}
```

Comment:
I think this example is actually fine, because no pointer to `x` is being returned.

Reply:
👍